gradle.properties

```diff
@@ -17,7 +17,7 @@ mockitoVersion=3.3.3
 pravegaVersion=0.12.0
```

Comment:
We need to update the pravega dependency as well to align the 0.13 version.

Reply:
Typically in this repo, we've left it to the officially released pravega version (0.12 at this time) until either:
- pravega releases a new release  0.13 (no snapshot)
- or somehow there is something in the pravega release that this library needs, in which case we'd take a 0.13-snapshot-xyz version.

That second bullet is extremely rare.  So I prefer to stick with the stable 0.12 until it's time.